components/SmoothLoadingImage.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  
+    <SmoothLoadingImageFadeTransition><div v-show="!show.showing"><SVGLoadingIcon /></div></SmoothLoadingImageFadeTransition>
+   <SmoothLoadingImageFadeTransition><img v-show="show.showing" :data-testid=testId :class="`object-${objectStyling} rounded-${borderRadius} w-${width} h-${height}`" :src="src" @load="showImg()"/></SmoothLoadingImageFadeTransition>
+
+</template>
+
+<script lang='ts' setup>
+  import { reactive, ref, watch, type Ref } from 'vue'
+  import SVGLoadingIcon from '~/assets/icons/loading.svg'
+
+  const props = defineProps({
+    imgSrc: {
+        type: String,
+        required: true,
+    },
+    testId: {
+        type: String,
+        required: true,
+    },
+    alt: {
+        type: String,
+        required: true,
+    },
+    objectStyling:{
+        type: String,
+        required: true,
+    },
+    height:{
+        type: String,
+        required: true,
+    },
+    width:{
+        type: String,
+        required: true,
+    },
+    borderRadius:{
+        type: String,
+        required: true,
+    },
+  })
+  
+  const src:Ref<string | undefined> = ref()
+  const show = reactive({showing: false});
+  const showImg = function () {
+    show.showing = true
+  }
+  
+  watch(
+    () => props.imgSrc,
+    () => {
+    src.value = props.imgSrc
+    show.showing = false
+  }, {immediate: true})
+  
+
+</script>
+
+
+<style scoped>
+  .loader {
```

Comment:
move these to tailwind style!  🐈 💨  (closest I can make to a tail-wind emoji XD)